src/LoaderController.res

```diff
@@ -359,6 +359,9 @@ let make = (~children, ~paymentMode, ~setIntegrateErrorError, ~logger, ~initTime
             dict->getJsonObjectFromDict("isReadyToPay")->JSON.Decode.bool->Option.getOr(false)
           )
         }
+        if dict->getDictIsSome("endpoint") {
+          ApiEndpoint.setApiEndPoint(dict->getString("endpoint", ""))
```

Comment:
Please check separately if endpoint is empty or not. In case it is empty, its better to keep apiEndPoint as undefined so it picks from GlobalVars